scripts/reports/generate-sbom.sh

```diff
@@ -29,16 +27,41 @@ function main() {
   enrich-report
 }
 
+# Create the report.
 function create-report() {
```

Comment:
Could this be renamed to be create-sbom-report?

Reply:
done, thanks

Reply:
I changed the file name for readability and clarity.